src/chapters/sql-part1/joins.rst

```diff
@@ -0,0 +1,111 @@
+Joins
+=====
+
+.. index:: ! join
+
+A **join** combines two tables into one result set.
+We can use joins when wanting to query two tables at the same time. 
+Whenever we join two tables, we have to specify the condition upon which the tables need to be joined.
+
+In SQL, there are four different types of joins:
+
+#. Inner Join
+#. Left Outer Join
+#. Right Outer Join
+#. Full Outer Join
+
+.. index:: ! inner join
+
+Inner Join
+----------
+
+Joining two tables on an inner join produces a result set that only includes the values that match in both tables.
+
+.. TODO: Add a diagram to demonstrate what result set looks like.
+
+.. admonition:: Example
+
+   If we use an inner join to combine ``JohnsonWedding`` and ``JohnsonVowRenewal`` in a query, we can see what guests are going to both the vow renewal and the wedding.
+
+   .. sourcecode:: mysql
+
+      SELECT LastName, FirstName
+      FROM JohnsonVowRenewal
+      INNER JOIN JohnsonWedding ON JohnsonVowRenewal.GuestID == JohnsonWedding.GuestID;
+
+   This query will give us a result set of the first and last names of the guests from the ``JohnsonVowRenewal`` table that are also in the ``JohnsonWedding`` table.
+
+Left Outer Join
+---------------
+
+Joining two tables on a left outer join gives us a result set will include all values in the left table and any matching records from the right table.
+
+.. TODO: Add a diagram to demonstrate what result set looks like.
+
+.. admonition:: Example 
+
+   If we use a left inner join to combine ``JohnsonWedding`` and ``JohnsonVowRenewal`` in a query, we can see what all of the guests that were invited to the wedding and any guests who were also invited to the vow renewal.
```

Comment:
I keep re-reading this sentence, but I still cannot quite picture what it is telling me.

Will we see all of the guests in the original wedding plus those who are expected at the vow renewal?